tests/ci/profiles/tf_cluster_profile.yml

```diff
@@ -48,7 +48,7 @@ profiles:
     compute_machine_type: "m5.2xlarge"
     proxy: true
     labeling: true
-    tagging: true
+    tagging: false
```

Comment:
why false?

Reply:
opsss, you are right. leftover from testing
fixed